### DISCUSSION TITLE
Is it possible to disable the theme for terminals?

### DISCUSSION THREAD

**Discussion by flatoy** [NvChad/NvChad#2492] [Q&A]
It seems like the theme breaks some formatting and most colors on logging and terminal tools. Of course I could just open up a new iterm tab, but I like the swiftness of toggling a floating terminal in nvim. 

Lazy git for example: 

Inside nvim:
<img width="1719" alt="Screenshot 2023-11-13 at 13 15 48" src="https://github.com/NvChad/NvChad/assets/11570271/33e5bb50-9a65-4d7b-9440-8510f81af03c">

In regular terminal:
<img width="1712" alt="Screenshot 2023-11-13 at 13 16 42" src="https://github.com/NvChad/NvChad/assets/11570271/10ebb98c-ff50-44aa-8e33-be0a790bd599">

**Comment by monok-robeto** [comment]
Any solution !?

**Reply by siduck** [reply]
@monok-robeto M.term field in chadrc, there's an option for this 

go read :h nvui
